docs/config-file-curations-yml.md

```diff
@@ -86,47 +86,31 @@ Where the list of available options for curations is defined in
 
 ## Command Line
 
-To use the `curations.yml` file put it to `$ORT_CONFIG_DIR/curations.yml` or pass it to the `--package-curations-file`
-option of the _analyzer_:
+To make ORT use the `curations.yml` file, put it to the default location of `$ORT_CONFIG_DIR/curations.yml` and then run
+the _analyzer_:
 
 ```bash
 cli/build/install/ort/bin/ort analyze
   -i [source-code-of-project-dir]
   -o [analyzer-output-dir]
-  --package-curations-file $ORT_CONFIG_DIR/curations.yml
 ```
 
-Alternatively specify a directory with multiple curation files using the `--package-curations-dir` to the _analyzer_:
-
-```bash
-cli/build/install/ort/bin/ort analyze
-  -i [source-code-of-project-dir]
-  -o [analyzer-output-dir]
-  --package-curations-dir $ORT_CONFIG_DIR/curations
-``` 
-
-ORT can use [ClearlyDefined](https://clearlydefined.io/) as a source for curated metadata. The preferred workflow is to
-use curations from ClearlyDefined, and to submit curations there. However, this is not always possible, for example in
-case of curations for organization internal packages. To support this workflow, ClearlyDefined can be enabled as the
-single source for curations or in combination with a `curations.yml` with the `--clearly-defined-curations` option of
-the analyzer:  
-
-```bash
-cli/build/install/ort/bin/ort analyze
-  -i [source-code-of-project-dir]
-  -o [analyzer-output-dir]
-  --package-curations-file $ORT_CONFIG_DIR/curations.yml
-  --clearly-defined-curations
-```
+Alternatively to a single file, curations may also be split across multiple files below a directory, by default
+`$ORT_CONFIG_DIR/curations`. Both file and directory package curation providers may also be configured as
+[FilePackageCurationProviders](../analyzer/src/main/kotlin/curation/FilePackageCurationProvider.kt) in
+`$ORT_CONFIG_DIR/config.yml`. Similarly, ORT can use [ClearlyDefined](https://clearlydefined.io/) and
```

Comment:
Would it make sense to somehow say what can be configured, e.g. using a different location?

...may be configured to use locations **different from the default ones**, see...

Reply:
> Would it make sense to somehow say what can be configured, e.g. using a different location?

Which type of provider are you referring to, only the `FilePackageCurationProviders`? Because for other, more can be configured (like credentials), but I didn't want to go into the details of the configuration options for each type here.

Reply:
Agree, it doesn't make sense to repeat these details. I thought that the point of the sentence, as-is, was to make clear that you can deviate from the default location. If that wasn't the case than my comment above comment does not really make sense.